Lib/multiprocessing/reduction.py

```diff
@@ -248,34 +273,61 @@ def _rebuild_socket(df, family, type, proto):
     register(socket.socket, _reduce_socket)
 
 
-class AbstractReducer(metaclass=ABCMeta):
+class AbstractReducer(metaclass=abc.ABCMeta):
     '''Abstract base class for use in implementing a Reduction class
     suitable for use in replacing the standard reduction mechanism
     used in multiprocessing.'''
-    ForkingPickler = ForkingPickler
-    register = register
-    dump = dump
     send_handle = send_handle
```

Comment:
None of the attributes defined between here and `__init__` appear to be used.

Reply:
Same comment as below.